inclusive-design/wecount.inclusivedesign.ca
# Improve the "Tags" page

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/227.

Improvements to the "Tags" page include:
1. Add the pagination;
2. Show proper page title;
3. News and views items show different layouts.

## Steps to test

* Test improvements 1 & 2:
1. Go to "views" -> click a test post for example "post 10" -> scroll to "tags" section and click the tag named "test post";
2. The tags page for "test post" is displayed.

**Expected behavior:** 
1. The pagination is showed at the bottom of the page;
2. The page title is "Tags (test post | Page 1);
3. click the pagination numbers to test other pages;
4. The page number in the page tile should adjust accordingly.

* Test improvements 3:
1. Go to "views" -> scroll to the pagination section and click page 11 -> click a post named "The Importance of Peripheral Vision" -> scroll to "tags" section and click the tag named "Smart Cities";
2. The tags page for "Smart Cities" is displayed.

**Expected behavior:** 
1. There are 3 items on this page. 1 is views item and the other two are news items;
2. Views items should display the default placeholder cover image and the author while news items don't display these.
